src/promptflow/promptflow/_core/connection_manager.py

```diff
@@ -44,6 +45,12 @@ def _build_connections(cls, _dict: Dict[str, dict]):
         for key, connection_dict in _dict.items():
             if isinstance(connection_dict, _Connection):
                 # support directly pass connection object to executor
+                scrubbed_secrets = connection_dict._get_scrubbed_secrets()
```

Comment:
@wangchao1230 shall we add instructions on how to fetch connection with secrets when this error happens?

Reply:
As offline synced, we won't add the usage in error message, maybe we'll add a readme for how to get connecion with/without secrets.